bitnami/metrics-server/templates/metric-server-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "metrics-server.fullname" . }}
+  labels:
+    app: {{ template "metrics-server.name" . }}
+    chart: {{ template "metrics-server.chart" . }}
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  ports:
+    - port: 8443
```

Comment:
this port should 443 to the outside. Otherwise, other resources will not fetch the metrics properly. I tested with this example: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/